src/main/java/org/jabref/logic/importer/WebFetchers.java

```diff
@@ -5,6 +5,10 @@
 import java.util.List;
 import java.util.Optional;
 
+import javafx.collections.FXCollections;
```

Comment:
Please no JavaFX imports. Our logic module must not have any imports from the gui. This blocks https://github.com/JabRef/jabref/issues/110

Why do our architecture tests not fail here?

Reply:
Should i refactor to standard library SortedSet?
In case Eclipse Collections is added as dependency later on,  ImmutableSortedSet can be used as drop in replacement.